datalad_deprecated/resources/website/assets/js/main.js

```diff
@@ -565,14 +600,48 @@ function directory(jQuery, md5) {
         else if (traverse.type === 'search')
           window.location.search = traverse.next;
       });
+
+      jQuery('table button').click(function(event) {
+        event.stopPropagation();
+        if (!jQuery(this).next().is(":visible")){
+          jQuery('.context-content:visible').hide();
```

Comment:
looks like identical line here and in `else` -- move up outside of `if`. Code duplication is evil!

Reply:
Oh wait, I do need both the lines, as the line after the hide() depends on the previous hide() statement (hide all others and show this one vs close everything including the currently open one).